man/man1/oci-kvm.1

```diff
@@ -88,14 +119,25 @@ option is not specified, an unused block device will be chosen arbitrarily. This
 For the
 .B create-pool
 command, the block device that will be used to build the storage pool.
+.IP "-N NETWORK_NAME, --network-name NETWORK_NAME"
+For the
+.B create-network
+and the
+.B delete-network
+commands, the name of the virtual (libvirt) network
```

Comment:
The name of the virtual (libvirt) network for the
.B create-network
and
.B delete-network
subcommands

Reply:
done